### ISSUE TITLE
choco --version includes warnings, breaks version parsing

### ISSUE THREAD

**Issue by mrhockeymonkey** [chocolatey/choco#1562]
### What You Are Seeing?
We use choco --version to parse the version of chocolatey that is running. However currently due to an expired licence a warning is also displayed that is stopping us from parsing the version. We then thought we could use the file version of the choco.exe itself but there was a mismatch. 

```powershell
choco -v
A license was found for a licensed version of Chocolatey, but is invalid:
 Expiration Date : 00/00/0000 00:00:00
0.10.8

(gi C:\ProgramData\chocolatey\bin\choco.exe).versioninfo.fileversion
0.10.5.0
```

### What is Expected?
a) the file version should match the version choco.exe claims to be
b) when running choco --version the only output should be the version. suppress all warnings and other output

Note the license isn't the issue here we are in the process of updating it but we now have to add a fix for this unexpected output. Not a huge issues nor a priority but it would be nice for future implementations to be able to trust this command.

**Comment by gep13** [comment]
@mrhockeymonkey for now, you should be able to use:

`(Get-Item C:\ProgramData\chocolatey\choco.exe).versioninfo.fileversion`

The choco.exe in the other location is actually a `shim` to the actual exe (although it should have the same version number I believe).

**Comment by gep13** [comment]
@mrhockeymonkey on a side note...

We recently released 0.10.11 of Chocolatey, which include an update for a reported CVE in the bundled version of 7zip (if you didn't see this announcement, you can sign up here: https://groups.google.com/forum/#!forum/chocolatey-announce).  We would recommend that you upgrade at your earliest convenience.

Details on the CVE can be found here:

https://groups.google.com/forum/#!topic/chocolatey-announce/0PwGifE23lg

**Comment by ferventcoder** [comment]
This is completed for Chocolatey v0.10.12.
